composer.json

```diff
@@ -18,7 +18,7 @@
   ],
   "require": {
     "php": "^7.0",
-    "phpunit/phpunit": "~6.0",
+    "phpunit/phpunit": ">6.0",
```

Comment:
Quite dangerous. Maybe it doesn't work with future versions of phpunit. I think "phpunit/phpunit": "~6.0||~7.0" is better here.